packages/create-plugin/src/cli.ts

```diff
@@ -2,8 +2,13 @@
 
 import { create } from 'create-create-app';
 import { resolve } from 'path';
+import fs from "fs";
+import { DownloaderHelper } from "node-downloader-helper";
+import decompress from "decompress"
 
 const templateRoot = resolve(__dirname, '..', 'templates');
+const modelsUrl = "https://github.com/HarlanHaller/3d-models-temp/archive/refs/heads/main.zip";
```

Comment:
I think instead of a separate repo being created maybe an assets folder can be added at the root of the https://github.com/frc-web-components/frc-web-components repo with a zip folder containing all the 3d models?

Reply:
The link now uses a zip stored in the root of my repo can very easily be changed to work with a file in the root of yours. The link and download name vars are the only things that need to be updated. NOTE: the folder created by the zip has to have the same name as the zip.

Reply:
Just added it here: https://github.com/frc-web-components/frc-web-components/tree/master/assets